Code/Network/RKHTTPRequestOperation.m

```diff
@@ -112,13 +112,6 @@ - (NSError *)error
 
 #pragma mark - NSURLConnectionDelegate methods
 
-- (void)connection:(NSURLConnection *)connection didReceiveAuthenticationChallenge:(NSURLAuthenticationChallenge *)challenge
```

Comment:
why was this removed ? Remember, RK supports back to iOS 5.1.1, which may well still need this method